CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+- Task becomes Complete and send Single worksheet
```

Comment:
This doesn't feel specific enough - which task?

Reply:
No worries, I have change this. Thanks Nick 👍